src/main/java/com/example/bank/account/TransferService.java

```diff
@@ -0,0 +1,57 @@
+package com.example.bank.account;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Slf4j
+@Getter
+public class TransferService {
+    public static void main(String[] args){
+        Account account = new Account();
+        List<Account> twoMyBankAccounts = account.createTwoMyBankAccounts();
+
+        Account polishAccount = new Account();
+        List<Account> twoPolishAccounts = account.createTwoPolishAccounts();
+
+        Account foreignAccount = new Account();
+        List<Account> twoForeignAccounts = account.createForeignAccounts();
+
+        TransferService myTransfer = new TransferService();
+        myTransfer.internalAndExternalTransfer(twoMyBankAccounts.get(0),twoMyBankAccounts.get(1),BigDecimal.valueOf(100L));
+    }
+    //TODO to może być jakiś interfejs w sumie; metoda abstrakcyjna; niezależna od banku i waluty
+    boolean internalAndExternalTransfer(Account firstAccount, Account secondAccount, BigDecimal amount){
+        if (firstAccount.getBalance().compareTo(amount) < 0 &&
```

Comment:
takie potezne ify moga byc juz metodami tak samo ten else. Ktorego moze nie byc bo jak nie wejdzie w if to wiadomo ze pojdzie w else